coalib/results/Diff.py

```diff
@@ -97,7 +97,7 @@ def _get_change(self, line_nr, min_line=1):
         if not isinstance(line_nr, int):
             raise TypeError('line_nr needs to be an integer.')
         if line_nr < min_line:
-            raise ValueError('The given line number is not allowed.')
+            raise IndexError('The given line number is not allowed.')
```

Comment:
This should ideally be in its own commit since we want atomic commits. But if there are no other reviews, we can ignore it for now. (Although next time, you should make commits atomic.)